### PR TITLE
feat: features section

### PR DESCRIPTION
**Pull request by krishguptadev**
- [x] Adds features section to the creators page!

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/91"></a>

### CODE REVIEW

File: src/components/Creators/Features.astro [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<section class="flex flex-col justify-space-around items-center px-10 lg:px-20 bg-gray-100 py-24 lg:py-40">
+  <h3 class="font-black font-display text-4xl w-full my-4">Features</h3>
+  <div class="grid grid-cols-1 ml-0 lg:grid-cols-2 gap-8 gap-x-0 w-auto text-2xl place-items-start w-full">
+    <div class="md:p-4 md:pl-0 ml-0 m-4">
+      <h3 class="font-bold font-display mb-4">Tokenization</h3>
```
Comment (krishguptadev) [inline src/components/Creators/Features.astro:5]:
```suggestion
      <h3 class="font-black font-display mb-4">Tokenization</h3>
```

File: src/components/Creators/Features.astro [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      <p class="text-base max-w-prose">Dev Protocol makes it easy to tokenize and authenticate the ownership of the creator’s work (e.g., GitHub repository for OSS developers), and “Property tokens” are issued by each work. It works like an “Equity”, automatically monetizing and distributing rewards to its holders.</p>
+    </div>
+    <div class="md:p-4 md:pl-0 ml-0 m-4">
+      <h3 class="font-bold font-display mb-4">Staking</h3>
```
Comment (krishguptadev) [inline src/components/Creators/Features.astro:9]:
```suggestion
      <h3 class="font-black font-display mb-4">Tokenization</h3>
```

File: src/components/Creators/Features.astro [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <p class="text-base max-w-prose">Staking is a mechanism that increases rewards through the temporary deposit of DEV token toward a property. By using staking to trade your business, you can bring in new kinds of subscriptions that are long-term and sustainable.</p>
+    </div>
+    <div class="md:p-4 md:pl-0 ml-0 m-4">
+      <h3 class="font-bold font-display mb-4">Market</h3>
```
Comment (krishguptadev) [inline src/components/Creators/Features.astro:13]:
```suggestion
      <h3 class="font-black font-display mb-4">Tokenization</h3>
```

File: src/components/Creators/Features.astro [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      <p class="text-base max-w-prose">Market is a contract to define the tokenizable assets in Dev Protocol, which can be added by community governance. It uses Khaos to add variety of assets in a very flexible way.</p>
+    </div>
+    <div class="md:p-4 md:pl-0 ml-0 m-4">
+      <h3 class="font-bold font-display mb-4">Khaos</h3>
```
Comment (krishguptadev) [inline src/components/Creators/Features.astro:17]:
```suggestion
      <h3 class="font-black font-display mb-4">Tokenization</h3>
```

File: src/components/Creators/Features.astro [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      <p class="text-base max-w-prose">Khaos is a very unique oracle service designed to bring Internet data into blockchains while keeping secret information, such as secret tokens, under wraps. It can prove the ownership of the work in a secure way.</p>
+    </div>
+    <div class="md:p-4 md:pl-0 ml-0 m-4">
+      <h3 class="font-bold font-display mb-4">DEV token</h3>
```
Comment (krishguptadev) [inline src/components/Creators/Features.astro:21]:
```suggestion
      <h3 class="font-black font-display mb-4">Tokenization</h3>
```

File: src/components/Creators/Features.astro [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<section class="flex flex-col justify-space-around items-center px-10 lg:px-20 bg-gray-100 py-24 lg:py-40">
+  <h3 class="font-black font-display text-4xl w-full my-4">Features</h3>
```
Comment (krishguptadev) [inline src/components/Creators/Features.astro:2]:
```suggestion
  <h3 class="font-black font-display text-6xl w-full my-4">Features</h3>
```

**Review by krishguptadev — COMMENTED** [review]
This is the most I can do without tampering with the consistency of the styles 👇

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by shuklaritvik06 — APPROVED** [review]

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Pull request merged**